runtime/v2/sdk/src/main/java/com/walmartlabs/concord/runtime/v2/sdk/SecretService.java

```diff
@@ -38,6 +47,53 @@
 
     String encryptString(String orgName, String projectName, String value) throws Exception;
 
+    @Value.Immutable
+    @Value.Style(jdkOnly = true)
+    interface SecretOperationResult {
```

Comment:
Maybe rename it ot `SecretCreationResult`? Or add a field similar to the Server's `OperationResult` enum? Not sure what's better. In theory, we might want to add methods to update or delete secrets as well and that's when we might want a separate field that indicates the type of operation that was performed.